### PR TITLE
Add an option to set the Content-Type of the response

### PR DESCRIPTION
**Pull request by shaiu**
1. add attribute content_type and set it
2. update README.md with Binary Response and Content Type sections

### CODE REVIEW

**Comment by shaiu** [comment]
@eliav-lavi I've added another attribute `content_type`
Also updated the README

**Comment by shaiu** [comment]
@eliav-lavi ping

File: app.rb [hunk 1]
```diff
@@ -158,7 +158,7 @@ def self.register_endpoint(endpoint:, sleep_time:)
     end
     method = METHODS[endpoint.verb]
     self.send(method, endpoint.path) do
-      content_type :json
+      content_type endpoint.content_type.empty? || endpoint.content_type == 'json' ? :json : endpoint.content_type
```
Comment (eliav-lavi) [inline app.rb]: Isn't this just
```suggestion
      content_type endpoint.content_type.empty? ? :json : endpoint.content_type
```

**Comment by eliav-lavi** [comment]
@shaiu apologies, I was away. Just a small comment if you may. And thanks for the documentation!

**Review by eliav-lavi — APPROVED** [review]

**Comment by eliav-lavi** [comment]
@shaiu thank you, merged, build image & [pushed](https://hub.docker.com/layers/eliavlavi/mox-server/0.5.4/images/sha256-8bfd21d9d5e9a29c7cd6ba50e6945eb80f3729d4d0bf6b6257930af06be25f10?context=explore).

**Pull request merged**